### ISSUE TITLE
BiocFileCache interventions

### ISSUE THREAD

**Issue by vjcitn** [mikelove/alevin2bioc#2]
not a bug but something to anticipate
```
library(tximeta)
se <- tximeta(files, type="alevin", alevinArgs=list(filterBarcodes=TRUE))
```
will, for users without the cache, generate two queries that need affirmative answers -- you might want to tweak the narrative to alert them to this

```tximeta needs a BiocFileCache directory to access and save TxDb objects.
Do you wish to use the default directory: '/home/rstudio/.cache/BiocFileCache'?
If not, a temporary directory that is specific to this R session will be used.

You can always change this directory later by running: setTximetaBFC()
Or enter [0] to exit and set this directory manually now. 

1: Yes (use default)
2: No (use temp)

Selection: 
Enter an item from the menu, or 0 to exit
Selection: 1
/home/rstudio/.cache/BiocFileCache
  does not exist, create directory? (yes/no): y
found matching transcriptome:
[ GENCODE - Homo sapiens - release 33 ]```

**Comment by mikelove** [comment]
Thanks Vince! Will address this in text now

**Comment by mikelove** [comment]
Resolved in 78efb216209316c34b19b68d45c790bac0f55ffd
